src/Wizard/WizardStep.jsx

```diff
@@ -116,7 +116,6 @@ WizardStep.propTypes = {
   help: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
   Component: PropTypes.oneOfType([PropTypes.func, PropTypes.instanceOf(React.Component)]).isRequired,
   componentProps: PropTypes.object,
-  nextButtonValue: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
   prevButtonValue: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
For consistency, would it make sense to move `prevButtonValue` into `Wizard` as well, while we're making this change, since that's also generated in the wizard?

Reply:
Sounds good. The button's already rendered in the Wizard so I'll remove these unused props. Thanks!